packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.styles.ts

```diff
@@ -28,17 +34,12 @@ const getBorderColor = (props) => {
   `;
 };
 
-const getBackgroundColor = (props) => {
+const getBackgroundColor = (props: TRichTextBodyStylesProps) => {
   if (props.isDisabled) {
     return css`
       background-color: ${vars.backgroundColorForInputWhenDisabled};
     `;
   }
-  if (props.isReadOnly) {
-    return css`
-      background-color: ${vars.backgroundColorForInputWhenReadonly};
-    `;
-  }
```

Comment:
Why is this removed?

Reply:
`backgroundColorForInputWhenReadonly` does not exist on from  `@commercetools-uikit/design-system` where `vars` is coming from.